src/azure-cli/azure/cli/command_modules/synapse/operations/workspace.py

```diff
@@ -15,19 +15,24 @@ def list_workspaces(cmd, client, resource_group_name=None):
 
 
 def create_workspace(cmd, client, resource_group_name, workspace_name, storage_account, file_system,
-                     sql_admin_login_user, sql_admin_login_password, location, enable_managed_virtual_network=None,
+                     sql_admin_login_user, sql_admin_login_password, location, key_name="default", key_identifier=None, enable_managed_virtual_network=None,
```

Comment:
just curious here:
As you know key identifier contains the key name, if customer only provides the key_identifier and maybe the key name in key identifier is not "default", what will happen?

Reply:
The key name is the display name at synapse workspace, different from the name of key-vault.

Reply:
> The key name is the display name at synapse workspace, different from the name of key-vault.

But the parameter is `--key-name`, maybe customer will think it is the key vault name, do you think it is a little confused?
@idear1203  How do you like?
I suggest to change the parameter name and give a clear description.

Reply:
I think the current name --key-name is fine. It it different from `--key-vault-name`

---
In reply to: [540663549](https://github.com/Azure/azure-cli/pull/16224#discussion_r540663549) [](ancestors = 540663549)